.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -0,0 +1,243 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+helper = null
+
+pipeline {
+    agent {
+        label 'kogito-operator-node && !built-in'
+    }
+
+    // Needed for local build
+    tools {
+        maven env.BUILD_MAVEN_TOOL
+        jdk env.BUILD_JDK_TOOL
+        go 'golang-1.19' // TODO to setup
+    }
+
+    options {
+        timeout(time: 24, unit: 'HOURS') // Give the day to be executed (due to the lock)
+    }
+
+    // parameters {
+    // For parameters, check into ./dsl/jobs.groovy file
+    // }
+
+    environment {
+        // Static env is defined into ./dsl/jobs.groovy file
+
+        KOGITO_CI_EMAIL_TO = credentials("${JENKINS_EMAIL_CREDS_ID}")
+
+        JAVA_HOME = "${GRAALVM_HOME}"
+
+        // Environment required by OLM tests when run from process
+        OP_TEST_CONTAINER_OPT = '-t'
+        OP_TEST_DEBUG = 1
+        OP_TEST_CONTAINER_TOOL = 'docker'
```

Comment:
We usually use podman by default on dev side (also on the other operators), isn't it better to switch ot it?

Reply:
sure will update this and try it